mfe/mfe_test.go

```diff
@@ -0,0 +1,1271 @@
+package mfe
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+)
+
+func ExampleMinimumFreeEnergy() {
+	mfe, _, _ := MinimumFreeEnergy("ACGAUCAGAGAUCAGAGCAUACGACAGCAG", "..((((...))))...((........))..", DefaultTemperature)
+	fmt.Println(mfe)
+	// Output:
+	// -2.9
+}
+
+func TestMinimumFreeEnergy(t *testing.T) {
+	test("ACGAUCAGAGAUCAGAGCAUACGACAGCAG",
```

Comment:
Can you comment what conditions these tests are testing for?

Reply:
What do you mean by conditions?